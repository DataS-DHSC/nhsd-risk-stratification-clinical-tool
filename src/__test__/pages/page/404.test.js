import React from 'react';
import { render } from '@testing-library/react';
import Error404 from '../../../pages/page/404';

const setup = () => render(<Error404 />);

describe('404 page', () => {
  it('should match snapshot', () => {
    const container = setup();
    expect(container.asFragment()).toMatchSnapshot();
  });
});
