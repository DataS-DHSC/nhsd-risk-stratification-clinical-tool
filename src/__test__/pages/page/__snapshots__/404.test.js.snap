// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`404 page should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="nhsuk-grid-row"
  >
    <div
      class="nhsuk-grid-column-full"
    >
      <h1
        id="error-heading"
      >
        Page not found
      </h1>
    </div>
    <div
      class="nhsuk-grid-column-two-thirds"
    >
      <p
        style="white-space: pre-wrap;"
      >
        If you entered a web address, make sure you entered it correctly.

If you pasted the web address, make sure you copied the entire address. 
      </p>
    </div>
  </div>
</DocumentFragment>
`;
